chcemvediet/apps/obligees/tests/test_models.py

```diff
@@ -1,369 +1,672 @@
 # vim: expandtab
 # -*- coding: utf-8 -*-
 import random
-import unittest
 
 from django.core.exceptions import ValidationError
 from django.db import IntegrityError
 from django.test import TestCase
+from django.utils.translation import ugettext_lazy as _
 
 from . import ObligeesTestCaseMixin
 from ..models import Obligee, HistoricalObligee
 
+
 class ObligeeModelTest(ObligeesTestCaseMixin, TestCase):
     u"""
     Tests ``Obligee`` and ``HistoricalObligee`` models.
     """
 
-    @unittest.skip(u'FIXME')
+    def test_official_name_field(self):
+        oblg = self._create_obligee(official_name=u'Agency Official')
+        self.assertEqual(oblg.official_name, u'Agency Official')
+
+    def test_official_name_field_default_value_if_omitted(self):
+        oblg = self._create_obligee(omit=[u'official_name'])
+        self.assertEqual(oblg.official_name, u'')
+
     def test_name_field(self):
         oblg = self._create_obligee(name=u'Agency')
         self.assertEqual(oblg.name, u'Agency')
 
-    @unittest.skip(u'FIXME')
```

Comment:
"# FIXME: Should be ``.get(name=value)``, but there are Obligees with duplicate names, yet." Je to stale aktualne, mozu existovat Obligee instancie s rovnakym nazvom? field `name` je definovany ako `models.CharField(max_length=255, unique=True,`

Reply:
@viliambalaz Tato otazka sa tyka tohto PR, ci niecoho z `test_forms.py`? Lebo nejak jej nerozumiem. Ak sa tyka `test_forms.py`, tak pls copy&pastni tu otazku tam a tu riesme iba `test_models`.